apps/minaintyg/src/components/Layout/Breadcrumbs.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react'
+import { Outlet, Route, RouterProvider, createMemoryRouter, createRoutesFromChildren } from 'react-router-dom'
+import { Breadcrumbs } from './Breadcrumbs'
+
+it('Should render breadcrumbs', () => {
```

Comment:
Smoke test can be added

Reply:
Not sure what that would be, please provide an example. This component currently has full test-coverage.

Reply:
It's described in the readme for the frontend project. If the readme includes wrong information on how the project does tests then it should be updated

Reply:
I see, I can update the readme. There is no need to test if a component crashes when there are other tests, if the component would crash the current test would fail.